prow/images/buildpack-node-chromium/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Node buildpack with chromium and its dependencies
+
+FROM eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181119-afd3fbd
```

Comment:
Change to same version as used in jobs: `eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181130-b28250b`

Reply:
done